invenio_cli/helpers/commands.py

```diff
@@ -395,6 +390,11 @@ def demo(self):
         self.docker_helper.execute_cli_command(
             project_shortname, 'invenio rdm-records demo')
 
+    def stop(self):
+        """Stops containers."""
+        self.docker_helper.stop_containers()
+        click.secho('Stopped containers', fg='green')
```

Comment:
ultra-mega-ocd, feel free to ignore: All the actions in the other commands are being logged before happening (that way if something goes wrong the user knows where, is not just "a hung up call". Same applies to destroy below.

Reply:
I don't think I fully understood what you meant... Could you explain a bit further?

Reply:
In other commands (e.g. when destroying indices or whatever), the message is `secho`ed before executing:

```
secho... # Tell the user it's gonna happen
docker_helper...  # Make it happen
```

Reply:
Hope it is better now :)